Qt.py

```diff
@@ -893,9 +893,14 @@ def load(self, uifile, *args, **kwargs):
                 etree = ElementTree()
                 etree.parse(uifile)
 
-                return Qt._QtUiTools.QUiLoader.load(
+                widget = Qt._QtUiTools.QUiLoader.load(
                     self, uifile, *args, **kwargs)
 
+                # Prevent RuntimeError in Nuke 9.0v9 with PySide 1.0.9
+                Qt.QtCompat._loadUi_temp = widget.parentWidget()
```

Comment:
Hm. Even if I delete it [here](https://github.com/fredrikaverpil/Qt.py/blob/d07a1e998261ad8006d8a8d52ef9533f2c96cbca/Qt.py#L938)?

    del Qt.QtCompat._loadUi_temp